packages/http-client-python/generator/pygen/codegen/templates/validation.py.jinja2

```diff
@@ -17,7 +34,7 @@ def api_version_validation(**kwargs):
             except AttributeError:
                 return func(*args, **kwargs)
 
-            if method_added_on > client_api_version:
+            if _index_with_default(method_added_on) > _index_with_default(client_api_version, -sys.maxsize):
```

Comment:
Adopt `sys.maxsize` instead of `float("inf") to avoid pyright error. Since sys.maxsize is large enough so there shall be no issue.
@iscai-msft for awareness.